packages/guitarix.vst/PKGBUILD

```
@@ -0,0 +1,69 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+pkgname=guitarix.vst
+pkgver=0.4
+pkgrel=2
+pkgdesc='A versatile (guitar) amplifier VST3 plugin'
+arch=(x86_64)
```

Let's try building for aarch64 as well ✊
```suggestion
arch=(aarch64 x86_64)
```